src/app/index.html

```diff
@@ -15,7 +15,8 @@
   pageTitle="Welcome"
   layout="container"
   showBreadcrumbs="true"
-  showTableOfContents="true">
+  showTableOfContents="true"
+  #stache>
```

Comment:
The reason I ask: I'm not sure what we gain, from just simply documenting `{{ config.skyux.appSettings.stache.example }}` (or even `{{ config.skyux.appSettings.example }}`).

The confusion of adding `#stache` to the `<stache>` tag isn't worth the small benefit of a simpler object chain, in my opinion.